src/components/Announcement.tsx

```diff
@@ -47,20 +47,16 @@ export default function () {
   const [animate, setAnimate] = useState(false)
   const location = useLocation()
 
-  if (announcementClosed) return null
-  const onPage = location.pathname === '/app'
+  const announcementPage = '/work'
```

Comment:
Why do we need this? It's this is next `flow`?

Reply:
I guess so& I've just pull updates from `work-proof`